backend/attendance/views.py

```diff
@@ -23,6 +27,33 @@ class LessonView(ModelViewSet):
     queryset = Lesson.objects.all()
     serializer_class = LessonSerializer
 
+    def create_from_recurrency(self, recurrent_datetime):
+        current_date = recurrent_datetime.start_datetime - timedelta(hours=3)
```

Comment:
Esse processamento é por causa que no banco salva como UTC? Confesso que não entendirrrr

Se for, eu acho que quando define a timezone no settings.py ele ja converte automático na hora de devolver

Mas posso estar enganado, se for o caso, daria pra adaptar no front tbm pra so mudar la